### ISSUE TITLE
BIG BUG Invulnerable gunners in moving vehicles

### ISSUE THREAD

**Issue by Elgin675** [acemod/ACE3#7132]
**Arma 3 Version:** 1.94.145977` (stable)
**CBA Version:** `3.12.0.190708` (stable)
**ACE3 Version:** `3.12.6` (stable)

**Mods:**
```
- CBA_A3
- ace
```

**Description:**
BIG BUG !!!
Gunners in open vehicles (cars, helicopters, etc.) are immortal when the vehicle is moving (fast). When the vehicle is stopped, the gunner becomes instant mortal.

The bug only appears during a multiplayer game with a minimum of two players, both local and dedicated servers and is therefore not reproducible in solo or alone in multiplayer mode.

The vehicle's speed change the damage taken by the gunner.
After tests we highlight that the vehicle need to go faster than 20km/h to give "protection" to the gunner.

**Steps to reproduce:**
- Shoot vehicle gunner (with at least two players on the game)

**Where did the issue occur?**
- Dedicated / Self-Hosted Multiplayer / Editor (Multiplayer)

**Additional information:**
- Please see these videos
https://youtu.be/1rdmO4urpSE?t=22m59s
https://youtu.be/1rdmO4urpSE?t=26m7s
https://youtu.be/1rdmO4urpSE?t=36m22s

The whole video is tests about this bug, testing in lastest version of ACE, for this video, the version was a bit changed to add pseudo on medical ui.

**Comment by PabstMirror** [comment]
I think this should be fixed in medical rewrite

**Comment by TheShinriel** [comment]
Certainly, we have not seen this problem for a long time ... maybe since the update of Contact.

Any idea about the release date for the medical rewrite (So eager to have it already :D)

**Comment by PabstMirror** [comment]
I may be thinking of a different issue then
I know old medical had some issues when riding in vehicles and taking damage from remote AI
but this may be something new

**Comment by dedmen** [comment]
This is a bug? I noticed this but I thought I'm just not hitting them...

**Comment by veteran29** [comment]
Seen something like this quite a few times (and died because of that). Always suspected that it's related to our big modpack. (encountered it way before contact release)

**Comment by Elgin675** [comment]
Indeed, we are very surprised that no one else has revealed this bug.

**Comment by Elgin675** [comment]
Some additional informations :

- The gunner is less resistant if he is naked / without ballistic protection ;
- The few injuries received are systematically and exclusively located on the arms and limited (a KSVK at 50 meters for a small avultion) ;
No other part of the body was affected.

- If the gunner aims with his weapon, he becomes even less vulnerable !

- When the gunner is hit, he screams loudly (in proportion to the size of the calibre used), but there is no damage.

**Comment by commy2** [comment]
Pretty sure this is just Flummi trying to fix vehicle crash damage.

No point in debugging if it can't be reproduced on master branch.

**Comment by commy2** [comment]
Still a thing after rewrite?

**Comment by ImplicitDeny** [comment]
It was not a thing for my team before 3.13, but it seems to be now. My teammates reported being unable (or having a really hard time) to take down a Quilin gunner while the vehicle is moving, despite firing from several different directions. This task was an easy one before, and something we are quite trained to and have done a lot of times during missions. But since 3.13, whatever the settings are (I know some of my teammates tried hard to find a balance but failed so far) it's extremely difficult to kill or knock them out.

This makes those light, open vehicles awfully dangerous now, because they are really mobile (very low crash/wallbang rate compared to heavier MRAP) and the minigun on top is hella precise when manned by an AI. Not a problem if we can have the initiative and have our sharpshooter one-shot the gunner, but nowadays it's not that easy apparently.

**Comment by commy2** [comment]
>It was not a thing for my team before 3.13, but it seems to be now.

Then it's something else.

**Comment by Elgin675** [comment]
> 
> 
> Still a thing after rewrite?

Unfortunately, I am currently unable to test it for now :(.
